<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title font="Georgia">NGS Resources and Cancer </title>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<link href="menu1.css" type="text/css" rel="stylesheet">
</head>
<body>
<h1 ><font color="#5b1356">Comprehensive Resources for Cancer NGS Data Analysis</font> </h1>
<div id="nav1">
<div id="nav">
<ul>
<li><a href="index.html">HOME</a></li>
<li><a href="ngstool.html">TOOLS</a>
<ul>

<li><a href="cancergenomics.html">Cancer&nbsp;Genomics</a></li>
<li><a href="cancertranscriptomics.html">Cancer&nbsp;Transcriptomics</a></li>
<li><a href="qualitycontrol.html">Quality&nbsp;Control</a></li>
<li><a href="Epigenomics.html">Cancer&nbsp;Epigenomics</a></li>
<li><a href="visualization.html">Cancer&nbsp;Genome&nbsp;visualization</a></li>



</ul>
</li>
<li><a href="pipelines.html">PIPELINES</a>
<ul>
<li><a href="ngsdapipelines.html">Data&nbsp;Analysis</a></li>
<li><a href="ngsvcpipelines.html">Variant&nbsp;Calling</a></li>
<li><a href="ngsqcpipelines.html">Quality&nbsp;Control</a></li></ul>
</li>

</li>
<li><a href="ngscancer.html">NGS and CANCER</a>
<ul>
<li><a href="ngsdataresource.html">Data&nbsp;Resources</a></li>
<li><a href="ngsresources.html">Cancer&nbsp;Databases</a></li>
<li><a href="ngsprojects.html">Cancer&nbsp;Projects</a></li>
<li><a href="pathways.html">Cancer&nbsp;Pathways</a></li>
</ul>
</li>
<li><a href="ngsfaq.html">FAQ</a></li>
<li><a href="ngsteam.html">TEAM</a></li>
<li><a href="ngscontactus.html">CONTACT US</a></li>

</ul>
</div></div><div id="list1">
<p><h2 align="center"><b>Tools in Cancer Genomics</b></h2></p><p align="justify" ><font color="#3f3a01"><b></b></<font></p>


      <h3 id="tab-title"><strong>Alignment</strong></h3>
      <ol>
        
        <li>
          <p align="left"><a href="http://sourceforge.net/apps/mediawiki/bfast/index.php?title=Main_Page" target="_blank">BFAST</a> :  An alignment tool for fast and accurate mapping of short reads to reference sequences </p>
        </li>
        <li>
          
          <p align="left"><a href="http://bio-bwa.sourceforge.net/" target="_blank">BWA</a> : A software package for mapping low-divergent sequences against a large reference genome, such as the human genome. </p></li>
              <li>
          <p align="left"><a href="http://bowtie-bio.sourceforge.net/index.shtml" target="_blank">Bowtie</a> : An ultrafast, memory-efficient short read aligner.</p>
        </li>
        <li>
          <p align="left"><a href="http://www.novocraft.com/" target="_blank">Novoalign/NovoalignCS</a> : An aligner for single-ended and paired-end reads from the Illumina Genome Analyser.  </p>
        </li>
        <li>
          <p align="left"><a href="http://maq.sourceforge.net/" target="_blank">MAQ</a> : Mapping and Assembly with Quality, builds assembly by mapping short reads to reference sequences.</p>

        </li>
        <li>
          <p align="left"><a href="http://compbio.cs.toronto.edu/shrimp/" target="_blank">SHRiMP</a> : A software package for aligning genomic reads against a target genome.</p>
        </li>
        <li>

          <p align="left"><a href="http://soap.genomics.org.cn/" target="_blank">SOAP2</a> : (Short Oligonucleotide
Analysis Package), a program for faster and efficient alignment for short oligonucleotide onto reference sequences.</p>

        <li>
          <p align="left"><a href="http://www.sanger.ac.uk/resources/software/ssaha2/" target="_blank">SSAHA2</a> :(Sequence Search and Alignment by Hashing Algorithm), a pairwise sequence alignment program designed for the efficient mapping of sequencing reads onto genomic reference sequences. </p>
        </li>
        <li>      <p align="left"><a href="http://www.irisa.fr/symbiose/projects/gassst/
" target="_blank">GASSST </a>  :   A Global Alignment Short Sequence Search Tool .  </p>      </li>
        
        <li>      <p align="left"><a href="http://pass.cribi.unipd.it/
" target="_blank">PASS </a>  :   A program to align short sequences .  </p>      </li>
        <li>      <p align="left"><a href="http://www.seqan.de/projects/MicroRazerS/
" target="_blank"> MicroRazerS</a>  : A rapid alignment of small RNA reads.  .  </p>      </li>
        <li>      <p align="left"><a href="http://www-personal.umich.edu/~jianghui/seqmap/
" target="_blank"> SeqMap</a>  :  A tool for mapping large amount of oligonucleotide to the genome .  </p>      </li>
        <li>      <p align="left"><a href="http://code.google.com/p/perm/
" target="_blank">PerM </a>  : An Efficient mapping of short sequences accomplished with periodic full sensitive spaced seeds  .  </p>      </li>
  
  
  
        </ol>
  <!--p>&nbsp;</p-->
					
					<h3 align="center"><strong> GENOMIC VARIATION DISCOVERY</strong></h3>
					
  <h3 id="tab-title"><strong>Mutation Calling</strong></h3>
<ol>


  <li>      <p align="left"><a href="http://aqua.unife.it/GAMES/" target="_blank">GAMES </a>  :  Identifies and annotates mutations in next-generation sequencing projects.  </p>      </li>
  <li>      <p align="left"><a href="http://omictools.com/sequencing/genome-resequencing/germline-calling/conan-snv-s532.html" target="_blank">CoNAn-SNV </a>  : A probabilistic framework for the discovery of single nucleotide variants in WGSS data.   </p>      </li>
  
  <li>      <p align="left"><a href="http://sourceforge.net/projects/lofreq/" target="_blank">LoFreq
 </a>  :  A fast and sensitive variant-caller for inferring single-nucleotide variants (SNVs) from high-throughput sequencing data.  </p>      </li>
  <li>

    <p align="left"><a href="http://www.broadinstitute.org/gsa/wiki/index.php/Unified_genotyper" target="_blank">GATK</a>: A multiple-sample, technology-aware SNV and indel caller.</p>

  </li>
      <li>

        <p align="left">
          <a href="http://compbio.bccrc.ca" target="_blank">JointSNVMix</a>: A probabilistic model for detection of somatic mutations in normal/tumour pair.
        </p>
      </li>
        <li>

          <p align="left"><a href="http://samtools.sourceforge.net/" target="_blank">SAMtools</a>: A set of utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer.</p>

        </li>
    <li>

          <p align="left"><a href="http://compbio.bccrc.ca/?page_id=204" target="_blank">SNVMix</a>: A tool for SNV calling based on probabilistic binomial mixture model.</p>

    </li>
        <li>

          <p align="left"><a href="http://soap.genomics.org.cn/soapsnp.html" target="_blank">SOAPsnp</a>: A tool for identifying SNVs by Beijing Genomics Institute (BGI).</p>

        </li>
          <li>

            <p align="left">
              <a href="ftp://strelka@ftp.illumina.com/" target="_blank">Strelka</a>: A tool for somatic small-variant calling from sequenced tumor-normal sample pairs.
            </p>

          </li>
          <li>

            <p align="left"><a href="http://genome.wustl.edu/software/somaticsniper" target="_blank">SomaticSniper</a>: A program to identify SNVs that are different between tumor and normal sample.</p>

      </li>
<li>

          <p align="left"><a href="http://varscan.sourceforge.net/" target="_blank">VarScan</a>: A platform-independent, technology-independent software tool for identifying SNVs, indels, and CNVs in massively parallel sequencing of individual and pooled samples.</p>
    </li></ol>
  <!--p>&nbsp;</p-->
    <h3 id="tab-title"><strong>Indel </strong></h3>
        <ol>
        <li>

          <p align="left"><a href="http://www.sanger.ac.uk/resources/software/dindel/" target="_blank">Dindel</a>: A program for calling small indels from short-read sequence data from Illumina platform.</p>

        </li>
        <li>

          <p align="left"><a href="https://trac.nbic.nl/pindel/" target="_blank">Pindel</a>: A tool for identifying indels and structural variants at single-based resolution from next-generation sequence data.</p>

        </li>
        <li>

          <p align="left"><a href="http://www.seqan.de/projects/" target="_blank">SplazerS</a>: A tool for detecting genomic indel variants with exact breakpoints in single- and paired-end sequencing.</p>

        </li>
  <li>      <p align="left"><a href="http://compbio.cs.toronto.edu/modil/" target="_blank"> MoDIL</a>  : Detecting INDEL Variation with Clone-end Sequencing - Resources and Source Code .  </p>      </li>
  
  <li>      <p align="left"><a href="https://code.google.com/p/pyrohmmvar/" target="_blank">PyroHMMvar </a>  :  A program to call short indels and SNPs for Ion Torrent and 454 data.    </p>      </li>
  
  
  
        </ol>
  <!--p>&nbsp;</p-->
    <h3 id="tab-title"><strong>Structural Variation</strong></h3>
        <ol>
  
  
  <li>      <p align="left"><a href="http://www.engr.uconn.edu/~jiz08001/svseq2.html" target="_blank"> svseq2</a>  : An improved approach for accurate and efficient calling of structural variations with low-coverage sequence data.   </p>      </li>
        <li>

          <p align="left"><a href="http://breakdancer.sourceforge.net/" target="_blank">BreakDancer</a>:  A tool for detecting five types of SVs (insertions, deletions, inversions, inter- and intra-chromosomal translocations) from next generation paired-end sequencing reads.</p>
        </li>
        <li><a href="ftp://ftp.stjude.org/pub/software/CREST/CREST.tgz" target="_blank">CREST</a>: A software  that uses the soft-clipped reads to directly map the breakpoints of SVs. </li>
        <li>

          <p align="left"><a href="http://code.google.com/p/gasv/" target="_blank">GASV</a>: A tool for identifying and comparing structural variants by computing intersections of breakpoint regions.</p>

        </li>
        <li>
          <p align="left">
            <a href="http://code.google.com/p/hydra-sv/" target="_blank">HYDRA</a>: A tool for detecting structural variants in both unique and duplicated genomic regions.
          </p>
        </li>
        <li>

          <p align="left"><a href="http://sv.gersteinlab.org/pemer/" target="_blank">PEMer</a>: A software package for detecting SVs from paired-end reads.</p>

        </li>
        <li>
          <p align="left">
            <a href="http://www.bioconductor.org/packages/2.10/bioc/html/R453Plus1Toolbox.html" target="_blank">R453Plus1Toolbox</a>: An R/Bioconductor package for the analysis of Roche 454 sequencing data.
          </p>
        </li>
        <li>

          <p align="left"><a href="http://svmerge.sourceforge.net//" target="_blank">SVMerge</a>: A tool for SVs analysis by integrating calls from several existing SV callers.</p>
        </li>
        <li>
          <p align="left">
            <a href="http://svdetect.sourceforge.net/Site/Home.html" target="_blank">SVDetect</a>: A tool for identifying structural variations from paired-end/mate pair data.
          </p>
        </li>
        <li>
          <p align="left">
            <a href="http://compbio.cs.sfu.ca/strvar.htm" target="_blank">VariationHunter</a>: An tool for identifying structural variations from paired-end WGS data.
          </p>
        </li>

<li>
          <p align="left">
            <a href="https://code.google.com/p/destruct/" target="_blank">destruct</a>: A software tool for identifying structural variation in tumour genomes from whole genome illumina sequencing.   </p>
        </li>
</ol>
  <!--p>&nbsp;</p-->

  <h3 id="tab-title"><strong>Copy Number Variation </strong></h3>
<ol>
       
        <li>

          <p align="left"><a href="https://dsgweb.wustl.edu/qunyuan/software/cmds/" target="_blank">CMDS</a>: A population-based method for recurrent CNVs analysis from multiple samples. </p>

        </li>
        <li>

          <p align="left"><a href="http://www.compbio.group.cam.ac.uk/Resources/CNAseg/CNAseg.rar" target="_blank">CNAseg</a>: A tool for Identifying CNVs in cancer from NGS data.</p>

        </li>
        <li>

          <p align="left"><a href="http://genome.wustl.edu/software/cnvhmm" target="_blank">cnvHMM</a>: A tool for CNVs analysis using Hidden Markov algorithm.</p>

        </li>
        <li>

          <p align="left"><a href="http://sv.gersteinlab.org/cnvnator/" target="_blank">CNVnator</a>: A tool for CNV discovery and genotyping from depth of read mapping.</p>

        </li>
        <li>

          <p align="left"><a href="http://bioinfo-out.curie.fr/projects/freec/" target="_blank">FREEC</a>: A tool for control-free CNVs detection using deep-sequencing data.</p>

        </li>
        <li>

          <p align="left"><a href="http://sourceforge.net/projects/rdxplorer/" target="_blank">RDXplorer</a>: A tool for CNVs detection in whole human genome sequence data using read depth coverage. </p>

        </li>
        <li>

          <p align="left"><a href="http://www.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&paper_id=182" target="_blank">SegSeq</a>: A tool for detecting CNVs from short sequence reads.</p>

        </li>
      <li>

        <p align="left"><a href="http://varscan.sourceforge.net/" target="_blank">VarScan</a>: A platform-independent, technology-independent software tool for identifying SNVs, indels, and CNVs in massively parallel sequencing of individual and pooled samples.</p>

      </li>
      
      <li>      <p align="left"><a href="http://sourceforge.net/projects/genseng/" target="_blank">GENSENG </a>  : A software detecting CNVs from NGS data.   </p>      </li>
      <li>      <p align="left"><a href="http://tiger.dbs.nus.edu.sg/cnv-seq/" target="_blank"> CNV-seq</a>  : A method for detecting DNA copy number variation (CNV) using high-throughput sequencing .  </p>      </li>
      <li>      <p align="left"><a href="http://mrcanavar.sourceforge.net/" target="_blank"> mrCaNaVaR </a>  : A copy number caller that analyzes the whole-genome next-generation sequence mapping read depth to discover large segmental duplications and deletions.  </p>      </li>
      
  <!--p>&nbsp;</p-->
</ol>
 <h3 id="tab-title"><strong>Mutation Effects</strong></h3>
     <ol><li>
               <p align="left"><a href="http://www.openbioinformatics.org/annovar/" target="_blank">ANNOVAR</a>: An efficient software tool to use update-to-date information to functionally annotate genetic variants detected from diverse genomes.</p>

        </li>
        <li>

          <p align="left"><a href="http://genetics.bwh.harvard.edu/pph2/" target="_blank">PolyPhen-2</a>: (Polymorphism Phenotyping v2) is a tool which predicts possible impact of an amino acid substitution on the structure and function of a human protein using straightforward physical and comparative considerations. </p>

        </li>
        <li>

          <p align="left"><a href="http://wiki.chasmsoftware.org" target="_blank">CHASM</a>: (Cancer-specific High-throughput Annotation of Somatic Mutations) is a method that predicts the functional significance of somatic missense mutations observed in the genomes of cancer cells.</p>
    <li>

          <p align="left"><a href="http://sift.jcvi.org/" target="_blank">SIFT</a>: predicts whether an amino acid substitution affects protein function.</p>
          </li>
          
          
          
          </ol>
           <!--p>&nbsp;</p-->

  <h3 id="tab-title"><strong> Assembly </strong></h3>
          <ol>
          	<li>      <p align="left"><a href="http://broadinstitute.org/software/allpaths-lg/blog/?page_id=12" target="_blank"> ALLPATHS-LG</a>  : High quality genome assembly from low cost data.  </p>      </li>
          	<li>      <p align="left"><a href="http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page" target="_blank"> Celera Assembler</a>  :  A de novo whole-genome shotgun (WGS) DNA sequence assembler.  </p>      </li>
          	<li>      <p align="left"><a href="http://www.geneious.com/workflows/genomics" target="_blank">Geneious </a>  : Software for analyzing both your high-throughput and Sanger sequencing data  .  </p>      </li>
          	<li>      <p align="left"><a href="http://locas.sourceforge.net" target="_blank"> LOCAS</a>  : A software to assemble short reads of next generation sequencing technologies at low coverage.  </p>      </li>
          	<li>      <p align="left"><a href="http://sourceforge.net/projects/contrail-bio/" target="_blank"> Contrail</a>  : A Hadoop based genome assembler for assembling large genomes in the clouds.  </p>      </li>
          	<li>      <p align="left"><a href="http://sourceforge.net/p/mira-assembler/wiki/Home" target="_blank"> MIRA</a>  : A whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads). .  </p>      </li>
          	
          	<li>      <p align="left"><a href="http://www.molecularevolution.org/software/genomics/velvet/" target="_blank"> Velvet</a>  : a de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454, developed by Daniel Zerbino and Ewan Birney at the European Bioinformatics Institute (EMBL-EBI), near Cambridge, in the United Kingdom .  </p>      </li>
          	
          	<li>      <p align="left"><a href="http://sourceforge.net/projects/congrpe" target="_blank"> CongrPE</a>  : a de novo assembly algorithm, named CongrPE, for Next-Generation Sequencing technology.  </p>      </li>
          	
          	<li>      <p align="left"><a href="http://lge.ibi.unicamp.br/zorro" target="_blank"> ZORRO</a>  : A hybrid sequencing technology assembler.  </p>      </li>
          	
          	<li>      <p align="left"><a href="http://bcgsc.ca/platform/bioinfo/software/abyss" target="_blank"> ABySS</a>  : Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler .  </p>      </li>
          </ol>
          
                   <!--p>&nbsp;</p-->
          
  <h3 id="tab-title"><strong> Annotation </strong></h3>
          <ol>
          	<li>      <p align="left"><a href="http://wannovar.usc.edu/" target="_blank"> wANNOVAR </a>  : Annotating genetic variants for personal genomes via the web.  </p>      </li>
          	<li>      <p align="left"><a href="http://www.openbioinformatics.org/annovar/" target="_blank"> ANNOVAR</a>  : An efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, as well as mouse, worm, fly, yeast and many others).  </p>      </li>
          	         	
          	<li>      <p align="left"><a href="http://www.svaproject.org/download.php" target="_blank"> SVA</a>  : (Sequence Variant Analyzer) Software to annotate,visualize and analyze the genetic variants identified through NGS.  </p>      </li>
          	<li>      <p align="left"><a href="http://gmod.org/wiki/WebApollo" target="_blank"> WebApollo</a>  : Browser based tool for visualization and editing of sequence annotations  .  </p>      </li>
          <li>
          	 <p align="left"><a href="http://www.well.ox.ac.uk/~kgaulton/chaos.shtml" target="_blank"> Chaos </a>  : A Perl-based system for annotation of variants identified in high-throughput sequencing experiments developed at the Wellcome Trust Centre for Human Genetics (WTCHG). </p>      </li> 
          </ol>
  
          
      </div>
</body>
</html>