<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title font="Georgia">NGS Resources and Cancer </title>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<link href="menu1.css" type="text/css" rel="stylesheet">
</head>
<body>
<h1 ><font color="#5b1356">Comprehensive Resources for Cancer NGS Data Analysis</font> </h1>
<div id="nav1">
<div id="nav">
<ul>
<li><a href="index.html">HOME</a></li>
<li><a href="ngstool.html">TOOLS</a>
<ul>

<li><a href="cancergenomics.html">Cancer&nbsp;Genomics</a></li>
<li><a href="cancertranscriptomics.html">Cancer&nbsp;Transcriptomics</a></li>
<li><a href="qualitycontrol.html">Quality&nbsp;Control</a></li>
<li><a href="Epigenomics.html">Cancer&nbsp;Epigenomics</a></li>
<li><a href="visualization.html">Cancer&nbsp;Genome&nbsp;visualization</a></li>



</ul>
</li>
<li><a href="pipelines.html">PIPELINES</a>
<ul>
<li><a href="ngsdapipelines.html">Data&nbsp;Analysis</a></li>
<li><a href="ngsvcpipelines.html">Variant&nbsp;Calling</a></li>
<li><a href="ngsqcpipelines.html">Quality&nbsp;Control</a></li></ul>
</li>

</li>
<li><a href="ngscancer.html">NGS and CANCER</a>
<ul>
<li><a href="ngsdataresource.html">Data&nbsp;Resources</a></li>
<li><a href="ngsresources.html">Cancer&nbsp;Databases</a></li>
<li><a href="ngsprojects.html">Cancer&nbsp;Projects</a></li>
<li><a href="pathways.html">Cancer&nbsp;Pathways</a></li>
</ul>
</li>
<li><a href="ngsfaq.html">FAQ</a></li>
<li><a href="ngsteam.html">TEAM</a></li>
<li><a href="ngscontactus.html">CONTACT US</a></li>

</ul>
</div></div><div id="list1">
<p><h2 align="center"><b>Tools in Cancer Transcriptomics</b></h2></p><p align="justify" ><font color="#3f3a01"><b></b></<font></p>
      <h3 id="tab-title"><strong>Spliced Alignment</strong></h3>
      <ol>
        
        <li>
          <p align="left"><a href="http://tophat.cbcb.umd.edu/" target="_blank">TopHat</a> :  A fast splice junction mapper for RNA-Seq reads. </p>
        </li>
        <li>      <p align="left"><a href="http://www.fml.mpg.de/raetsch/projects/qpalma
" target="_blank">QPALMA </a>  : A tool for Optimal Spliced Alignments of Short Sequence Reads .  </p>      </li>
    
        <li>
          
          <p align="left"><a href="http://www.netlab.uky.edu/p/bioinfo/MapSplice" target="_blank">MapSplice</a> : A tool for accurate mapping of RNA-seq reads for splice junction discovery . </p></li>
              <li>
          <p align="left"><a href="http://www.stanford.edu/group/wonglab/SpliceMap/" target="_blank">SpliceMap</a> : A de novo splice junction discovery and alignment tool.</p>
        </li>
        <li>
          <p align="left"><a href="http://research-pub.gene.com/gmap/" target="_blank">GMAP</a> : A genomic mapping and alignment program for mRNA and EST sequences .  </p>
        </li>
        <li>
          <p align="left"><a href="http://gingeraslab.cshl.edu/STAR/" target="_blank">STAR</a> : (Spliced Transcripts Alignment to a Reference) Aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.</p>

        </li>
        
          
        <li>        <p align="left"><a href="http://soap.genomics.org.cn/soapsplice.html" target="_blank">SOAPsplice</a> :  A tool for genome-wide ab initio detection of splice junction sites from RNA-Seq, a method using new generation sequencing technologies to sequence the messenger RNA.  </p>        </li>
        <li>        <p align="left"><a href="http://mocklerlab.org/tools/1" target="_blank">Supersplat</a> :  An application for discovering potential splice junctions in high throughput sequencing (HTS) data. </p>        </li>
        </ol>
  <!--p>&nbsp;</p-->
  
      <h3 id="tab-title"><strong>Differential expression</strong></h3>
      <ol>

        <li>
          <p align="left"><a href="http://cufflinks.cbcb.umd.edu/" target="_blank">CuffDiff</a> :assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples. </p>
        </li>
        
 <li>
          <p align="left"><a href="http://www.bioconductor.org/packages/2.11/bioc/html/edgeR.html" target="_blank">EdgeR</a> : (Empirical analysis of digital gene expression data in R) tool for Differential expression analysis of RNA-seq and digital gene expression profiles with biological replication. </p>
        </li>
        
  <li>
          <p align="left"><a href="http://www-huber.embl.de/users/anders/DESeq/" target="_blank">DESeq</a> :An R package to analyse count data from high-throughput sequencing assays such as RNA-Seq and test for differential expression. </p>
        </li>
        
  <li>
          <p align="left"><a href="http://bowtie-bio.sourceforge.net/myrna/index.shtml" target="_blank">Myrna</a> : cloud computing tool for calculating differential gene expression in large RNA-seq datasets. </p>
        </li>
        </ol>

 <!--p>&nbsp;</p-->
        
  		<h3 id="tab-title"><strong>Alternative Splicing</strong></h3>
      <ol>

   <li>
          <p align="left"><a href="http://cufflinks.cbcb.umd.edu/" target="_blank">CuffDiff</a> :assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples. </p>
        </li>
       
  
  <li>
          <p align="left"><a href="http://genes.mit.edu/burgelab/miso/" target="_blank"> MISO </a> : (Mixture of Isoforms) A probabilistic framework that quantitates the expression level of alternatively spliced genes from RNA-Seq data, and identifies differentially regulated isoforms or exons across samples. </p>
        </li>
        
  <li>
          <p align="left"><a href="http://bioconductor.org/packages/release/bioc/html/DEXSeq.html" target="_blank"> DEXseq</a> : Package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. </p>
        </li>
 				
  <li>
          <p align="left"><a href="http://www.alexaplatform.org/alexa_seq/" target="_blank"> Alexa-seq </a> : Method for using massively parallel paired-end transcriptome sequencing for 'alternative expression analysis'. </p>
        </li>
 <li>        <p align="left"><a href="http://sourceforge.net/projects/soapdenovotrans/" target="_blank">SOAPdenovo-Trans</a> : A de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts.  </p>        </li>
        
        </ol>
  <!--p>&nbsp;</p-->
  		<h3 id="tab-title"><strong>Gene Fusion</strong></h3>
  	<ol>
  	 <li>
      <p align="left"><a href="http://sourceforge.net/apps/mediawiki/defuse/index.php?title=Main_Page" target="_blank">defuse</a>: A software package for discovering gene fusion from  RNA-Seq data.
      </p>
        </li>
        <li>
          <p align="left">
            <a href="http://www.ilte-cml.org/FusionAnalyser/" target="_blank">FusionAnalyser</a>: A software to read paired reads mapping to different genes (Bridge reads),  generated through high-throughput whole transcriptome sequencing.
          </p>
        </li>
        <li>
          <p align="left"><a href="http://bioen-compbio.bioen.illinois.edu/FusionHunter/" target="_blank">FusionHunter</a>: A tool to identify fusion  transcripts from transcriptional analysis of paired-end RNA-seq. </p>
        </li>
        <li>
          <p align="left"><a href="http://www.omicsoft.com/fusionmap/" target="_blank">FusionMap</a>:  An efficient fusion aligner which aligns reads spanning fusion junctions directly to the genome without prior knowledge of potential fusion regions.
          </p>
        </li>
        <li>
      <p align="left"><a href="http://archive.gersteinlab.org/proj/rnaseq/fusionseq/" target="_blank">FusionSeq</a>:  A tool to identify fusion transcripts from paired-end RNA-sequencing.
      </p>
      </li>				
	
	
       <li>
          <p align="left"><a href="http://soap.genomics.org.cn/SOAPfusion.html" target="_blank"> SOAPfusion </a> : A novel tool for fusion discovery with paired-end RNA-Seq reads . </p>
        </li>
       <li>
          <p align="left"><a href="http://ccb.jhu.edu/software/tophat/index.shtml" target="_blank"> TopHat-Fusion </a> : Enhanced version of TopHat with the ability to align reads across fusion points, which results from the breakage and re-joining of two different chromosomes, or from rearrangements within a chromosome.
. </p>
        </li>
 
             </ol>				
      <!--p>&nbsp;</p-->
      </div>
</body>
</html>