<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title font="Georgia">NGS Resources and Cancer </title>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<link href="menu1.css" type="text/css" rel="stylesheet">
</head>
<body>
<h1 ><font color="#5b1356">Comprehensive Resources for Cancer NGS Data Analysis</font> </h1>
<div id="nav1">
<div id="nav">
<ul>
<li><a href="index.html">HOME</a></li>
<li><a href="ngstool.html">TOOLS</a>
<ul>

<li><a href="cancergenomics.html">Cancer&nbsp;Genomics</a></li>
<li><a href="cancertranscriptomics.html">Cancer&nbsp;Transcriptomics</a></li>
<li><a href="qualitycontrol.html">Quality&nbsp;Control</a></li>
<li><a href="Epigenomics.html">Cancer&nbsp;Epigenomics</a></li>
<li><a href="visualization.html">Cancer&nbsp;Genome&nbsp;visualization</a></li>



</ul>
</li>
<li><a href="pipelines.html">PIPELINES</a>
<ul>
<li><a href="ngsdapipelines.html">Data&nbsp;Analysis</a></li>
<li><a href="ngsvcpipelines.html">Variant&nbsp;Calling</a></li>
<li><a href="ngsqcpipelines.html">Quality&nbsp;Control</a></li></ul>
</li>

</li>
<li><a href="ngscancer.html">NGS and CANCER</a>
<ul>
<li><a href="ngsdataresource.html">Data&nbsp;Resources</a></li>
<li><a href="ngsresources.html">Cancer&nbsp;Databases</a></li>
<li><a href="ngsprojects.html">Cancer&nbsp;Projects</a></li>
<li><a href="pathways.html">Cancer&nbsp;Pathways</a></li>
</ul>
</li>
<li><a href="ngssearch.html">SEARCH</a></li>
<li><a href="ngsfaq.html">FAQ</a></li>
<li><a href="ngsteam.html">TEAM</a></li>
<li><a href="ngscontactus.html">CONTACT US</a></li>

</ul>
</div></div><div id="list1">
<p><h2 align="center"><b>Tools in Cancer Epigenomics</b></h2></p><p align="justify" ><font color="#3f3a01"><b></b></<font></p>
      <h3 id="tab-title"><strong>ChIP Seq</strong></h3>
      <ol>
        <li><p align="left"><a href="http://liulab.dfci.harvard.edu/MACS" target="_blank">MACS</a> : Model-based Analysis of ChIP-Seq (MACS) on short reads sequencers such as Genome Analyzer (Illumina / Solexa).</p>    </li>
        <li><p align="left"><a href="http://info.gersteinlab.org/PeakSeq" target="_blank">Peakseq</a> : program for identifying and ranking peak regions in ChIP-Seq experiments. </p>    </li>
        <li><p align="left"><a href="https://urgi.versailles.inra.fr/Tools/S-Mart" target="_blank">S-Mart</a> : An intuitive and lightweight tool, performing several tasks that are usually required during the analysis of mapped RNA-Seq and ChIP-Seq reads, including data selection and data visualization.</p>    </li>
        <li><p align="left"><a href="http://home.gwu.edu/~wpeng/Software.htm" target="_blank">SICER</a> : Package for identification of enriched domains from histone modification ChIP-Seq data.</p>    </li>
        <li><p align="left"><a href="http://meme.nbcr.net/meme/cgi-bin/meme-chip.cgi" target="_blank">MEME-ChIP</a> : Tool designed especially for discovering motifs in LARGE (50MB maximum) sets of short (around 500bp) DNA sequences centered on locations of interest such as those produced by ChIP-seq experiments. </p>    </li>
        <li><p align="left"><a href="http://cgs.csail.mit.edu/onePageGem/" target="_blank">GEM</a> : A Java software for studying protein-DNA interaction using ChIP-seq/ChIP-exo data.</p>    </li>
        <li><p align="left"><a href="http://meme.nbcr.net/meme/doc/dreme.html" target="_blank">DREME </a> : A tool for motif discovery in transcription factor ChIP-seq data .</p>    </li>
        </ol>
  <!--p>&nbsp;</p-->
  
      <h3 id="tab-title"><strong>Bisulphite Seq</strong></h3>
      <ol>
        
        <li>          <p align="left"><a href="http://epigenome.usc.edu/publicationdata/bissnp2011/" target="_blank">Bis-SNP</a> : Bisulfite Space genotyper and Methylation caller. </p>        </li>
        <li>          <p align="left"><a href="https://code.google.com/p/bsmap" target="_blank">Bsmap</a> : A short reads mapping software for bisulfite sequencing reads. </p>        </li>
        <li>          <p align="left"><a href="http://compbio.cs.ucr.edu/brat/" target="_blank">BRAT</a> : An accurate and efficient tool for mapping short bisulfite-treated reads obtained from the Solexa-Illumina Genome Analyzer. </p>        </li>
        <li>          <p align="left"><a href="http://code.google.com/p/batmeth/" target="_blank">BatMeth</a> : Improved mapper for bisulfite sequencing reads on DNA methylation. </p>        </li>
        <li>          <p align="left"><a href="http://code.google.com/p/bsolana" target="_blank">B-SOLANA</a> : A fast and accurate all-in-one approach including alignment and methylation calling of two-base encoding (colorspace) bisulfite sequencing data. </p>        </li>
        <li>          <p align="left"><a href="http://pass.cribi.unipd.it/cgi-bin/pass.pl?action=Download" target="_blank">PASS-bis</a> : A bisulfite aligner suitable for whole methylome analysis of Illumina and SOLiD reads. </p>        </li>
        <li>          <p align="left"><a href="http://www.bioinformatics.babraham.ac.uk/projects/bismark/" target="_blank">Bismark</a> : A tool to map bisulfite converted sequence reads and determine cytosine methylation states. </p>        </li>
        <li>          <p align="left"><a href="http://katahdin.mssm.edu/kismeth/revpage.pl" target="_blank">Kismeth</a> : An online tool for analysis of bisulphite treated reads. </p>        </li>
        <li>          <p align="left"><a href="http://pellegrini.mcdb.ucla.edu/BS_Seeker/BS_Seeker.html" target="_blank">BS Seeker </a> : Performs accurate and fast mapping of bisulfite-treated short reads . </p>        </li>
        
           </ol>				
      <!--p>&nbsp;</p-->
      <h3 id="tab-title"><strong>Methylation</strong></h3>
      <ol>
        
        <li><p align="left"><a href="http://bioinfo2.ugr.es/NGSmethPipe/" target="_blank">NGSmethPipe</a> : A tool to generate high-quality methylation maps. </p>    </li>
        <li><p align="left"><a href="https://github.com/BenLangmead/bsmooth-align" target="_blank">BSmooth-align</a> : A tool for alignment of bisulfite sequence reads and tabulation of read-level methylation measurements. </p>    </li>
        <li><p align="left"><a href="https://code.google.com/p/methylkit/" target="_blank">Methylkit</a> : An R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. </p>    </li>
        <li><p align="left"><a href="http://www.bioconductor.org/packages/release/bioc/html/methylumi.html" target="_blank">Methylumi</a> : This package provides classes for holding and manipulating Illumina methylation data.

</p>    </li>
        <li><p align="left"><a href="https://github.com/brentp/methylcode" target="_blank">MethylCoder</a> : Performs methylation analysis in single base pair resolution.</p>    </li>
           </ol>		
           <!--p>&nbsp;</p-->		
      
      </div>
</body>
</html>