<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title font="Georgia">NGS Resources and Cancer </title>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<meta name=”description” content=”Comprehensive Resources for Cancer NGS Data Analysis (CRCDA)”/>
<meta name=”keywords” content=”CRCDA, NGS, Cancer and NGS, NGS data analysis, CRCDA”/>
<link href="menu1.css" type="text/css" rel="stylesheet">
</head>
<body>
<h1 ><font color="#5b1356">Comprehensive Resources for Cancer NGS Data Analysis</font> </h1>
<div id="nav1">
<div id="nav">
<ul>
<li><a href="index.html">HOME</a></li>
<li><a href="ngstool.html">TOOLS</a>
<ul>

<li><a href="cancergenomics.html">Cancer&nbsp;Genomics</a></li>
<li><a href="cancertranscriptomics.html">Cancer&nbsp;Transcriptomics</a></li>
<li><a href="qualitycontrol.html">Quality&nbsp;Control</a></li>
<li><a href="Epigenomics.html">Cancer&nbsp;Epigenomics</a></li>
<li><a href="visualization.html">Cancer&nbsp;Genome&nbsp;visualization</a></li>



</ul>
</li>
<li><a href="pipelines.html">PIPELINES</a>
<ul>
<li><a href="ngsdapipelines.html">Data&nbsp;Analysis</a></li>
<li><a href="ngsvcpipelines.html">Variant&nbsp;Calling</a></li>
<li><a href="ngsqcpipelines.html">Quality&nbsp;Control</a></li></ul>
</li>

</li>
<li><a href="ngscancer.html">NGS and CANCER</a>
<ul>
<li><a href="ngsdataresource.html">Data&nbsp;Resources</a></li>
<li><a href="ngsresources.html">Cancer&nbsp;Databases</a></li>
<li><a href="ngsprojects.html">Cancer&nbsp;Projects</a></li>
<li><a href="pathways.html">Cancer&nbsp;Pathways</a></li>
</ul>
</li>
<li><a href="ngsfaq.html">FAQ</a></li>
<li><a href="ngsteam.html">TEAM</a></li>
<li><a href="ngscontactus.html">CONTACT US</a></li>

</ul>
</div></div><div id="list1">
<p><h2 align="center"><b>Pipelines for Variant Calling and Quality Control</b></h2></p><p align="justify" ><font color="#3f3a01"><b></b></<font></p>
      
  
      <h3 id="tab-title"><strong>Variant Calling</strong></h3>
      <ol>
        <li>      <p align="left"><a href="http://bioconductor.org/packages/2.12/bioc/html/cn.mops.html" target="_blank">cn.mops </a>  : A data processing pipeline for copy number variations and aberrations (CNVs and CNAs) from next generation sequencing (NGS) data.  .  </p>      </li>
        <li>
          <p align="left"><a href="https://bcbio-nextgen.readthedocs.org/en/latest/contents/pipelines.html" target="_blank">bcbio-nextgen</a> : bcbio-nextgen implements configurable best-practice pipelines for SNP and small indel calling. </p>
        </li>
        
        <li>      <p align="left"><a href="http://ingap.sourceforge.net/" target="_blank"> inGAP </a>  : An integrative next-generation genome analysis pipeline (inGAP), which employed a Bayesian principle to detect single nucleotide polymorphisms (SNPs), small insertion/deletions (indels).  </p>      </li>
        <li>      <p align="left"><a href="http://genomics.princeton.edu/AndolfattoLab/MSG.html" target="_blank"> MSG</a>  : (Multiplexed shotgun genotyping) A pipeline of scripts to assign ancestry to genomic segments using next-gen sequence data.   .  </p>      </li>
        <li>      <p align="left"><a href="https://github.com/cc2qe/speedseq" target="_blank">speedseq </a>  : A lightweight, flexible, and open source pipeline that identifies genomic variation (single nucleotide variants (SNVs), indels, and structural variants (SVs)).   </p>      </li>
           
        
        <li>      <p align="left"><a href="http://biostat.mc.vanderbilt.edu/wiki/Main/ASAP" target="_blank"> ASAP</a>  : An open source framework for building and executing a pipeline to preprocess next generation sequence data and variant calls.  </p>      </li>   
           
           </ol>					
      
      </div>
</body>
</html>
