<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title font="Georgia">NGS Resources and Cancer </title>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
 <meta name=”description” content=”Comprehensive Resources for Cancer NGS Data Analysis (CRCDA)”/>
<meta name=”keywords” content=”CRCDA, NGS, Cancer and NGS, NGS data analysis, CRCDA”/>
<link href="menu1.css" type="text/css" rel="stylesheet">
</head>
<body>
<h1 ><font color="#5b1356">Comprehensive Resources for Cancer NGS Data Analysis</font> </h1>
<div id="nav1">
<div id="nav">
<ul>
<li><a href="index.html">HOME</a></li>
<li><a href="ngstool.html">TOOLS</a>
<ul>

<li><a href="cancergenomics.html">Cancer&nbsp;Genomics</a></li>
<li><a href="cancertranscriptomics.html">Cancer&nbsp;Transcriptomics</a></li>
<li><a href="qualitycontrol.html">Quality&nbsp;Control</a></li>
<li><a href="Epigenomics.html">Cancer&nbsp;Epigenomics</a></li>
<li><a href="visualization.html">Cancer&nbsp;Genome&nbsp;visualization</a></li>



</ul>
</li>
<li><a href="pipelines.html">PIPELINES</a>
<ul>
<li><a href="ngsdapipelines.html">Data&nbsp;Analysis</a></li>
<li><a href="ngsvcpipelines.html">Variant&nbsp;Calling</a></li>
<li><a href="ngsqcpipelines.html">Quality&nbsp;Control</a></li></ul>
</li>

</li>
<li><a href="ngscancer.html">NGS and CANCER</a>
<ul>
<li><a href="ngsdataresource.html">Data&nbsp;Resources</a></li>
<li><a href="ngsresources.html">Cancer&nbsp;Databases</a></li>
<li><a href="ngsprojects.html">Cancer&nbsp;Projects</a></li>
<li><a href="pathways.html">Cancer&nbsp;Pathways</a></li>
</ul>
</li>
<li><a href="ngsfaq.html">FAQ</a></li>
<li><a href="ngsteam.html">TEAM</a></li>
<li><a href="ngscontactus.html">CONTACT US</a></li>

</ul>
</div></div>
<div id="list1">
<p><h2 align="center"><b>Quality Control Tools</b></h2></p><p align="justify" ><font color="#3f3a01"><b></b></<font></p>
      <h3 id="tab-title"><strong> Error detection and correction </strong></h3>
      <ol>
      <li>      <p align="left"><a href="http://shrec-ec.sourceforge.net/" target="_blank">SHREC </a> : (Short Read Error Correction) A bioinformatics tool for error correction of HTS read data.</p>      </li>
      <li>      <p align="left"><a href="http://genome.gsc.riken.jp/osc/english/dataresource/" target="_blank"> TagDust</a> :  A Program to eliminate artifactual reads from next-generation sequencing data sets.      </p>      </li>
      <li>      <p align="left"><a href="http://www.ebi.ac.uk/goldman-srv/AYB/" target="_blank">AYB </a> : A base caller for the Illumina Genome Analyzer, using an explicit statistical model of how errors occur during sequencing to produce more accurate reads from the raw intensity data.      </p>      </li>
      <li>      <p align="left"><a href=" http://www.cs.berkeley.edu/~yss/bayescall/" target="_blank">BayesCall </a> : A model-based basecalling algorithm for high-throughput short-read sequencing      </p>      </li>
      <li>      <p align="left"><a href="https://bioinf.eva.mpg.de/Ibis/" target="_blank"> Ibis </a>  : (Improved base identification system), is an accurate, fast and easy-to-use base caller for the Illumina sequencing system, which significantly reduces the error rate and increases the output of usable reads.     </p>      </li>
      <li>      <p align="left"><a href="http://sourceforge.net/projects/swiftng/" target="_blank">Swift </a> : Open source primary data analysis for next-gen DNA sequencers.
      </p>      </li>
      <li>      <p align="left"><a href="http://www.genome.umd.edu/quorum.html" target="_blank"> QuorUM </a>  : (Quality Optimized Reads from the University of Maryland) An error corrector for Illumina reads.</p>      </li>
      <li>      <p align="left"><a href="http://www.csd.uwo.ca/~ilie/HiTEC/" target="_blank">HiTEC </a> : Accurate error correction in high-throughput sequencing data.      </p>      </li>
      <li>      <p align="left"><a href="http://musket.sourceforge.net/homepage.htm#latest" target="_blank">Musket </a> : (multistage k-mer spectrum based corrector) An efficient multistage k-mer based corrector for Illumina short read data.      </p>      </li>
      <li>      <p align="left"><a href="http://uc-echo.sourceforge.net/" target="_blank">ECHO </a> : An error correction algorithm designed for short-reads from next-generation sequencing platforms such as Illumina's Genome Analyzer II.       </p>      </li>
      <li>      <p align="left"><a href="http://sourceforge.net/projects/trowel-ec/" target="_blank"> Trowel </a> : An error correction module for Illumina sequencing reads, which is based on the k-mer spectrum approach.       </p>      </li>
      <li>      <p align="left"><a href="http://aluru-sun.ece.iastate.edu/doku.php?id=reptile" target="_blank">Reptile </a> : A software developed in C++ for correcting sequencing errors in short reads from next-gen sequencing platforms.       </p>      </li>
      <li>      <p align="left"><a href="http://sourceforge.net/projects/hector454/" target="_blank">HECTOR </a> : A parallel multistage k-hopo spectrum based homopolymer-length error corrector for 454 sequencing data.      </p>      </li>
      <li>      <p align="left"><a href="http://decgpu.sourceforge.net/homepage.htm#latest" target="_blank">DecGPU  </a> : (DecGPU - Distributed error correction on CPU/GPU clusters) Parallel and distributed error correction algorithm for high-throughput short reads using CUDA and MPI parallel programming models.     </p>      </li>
      <li>      <p align="left"><a href="http://www.cs.helsinki.fi/u/lmsalmel/hybrid-shrec/" target="_blank">Hybrid SHREC </a>  : An error correction algorithm for correcting reads from various DNA sequencing platforms.    </p>      </li>
      
      <li>      <p align="left"><a href="http://sourceforge.net/projects/htqc/" target="_blank">HTQC </a>  : A toolkit including statistics tool for illumina high-throughput sequencing data, and filtration tools for sequence quality, length, tail quality, etc.  </p>      </li>
      <li>      <p align="left"><a href="http://www.computationalbioenergy.org/qc-chain.html" target="_blank">QC-Chain </a>  : A package of quality control tools for next generation sequencing (NGS) data, consisting of both raw reads quality evaluation and de novo contamination screening, which could identify all possible contamination sequences.   </p>      </li>
      <li>      <p align="left"><a href="http://www.nipgr.res.in/ngsqctoolkit.html" target="_blank">NGS QC Toolkit </a>  : A toolkit for the quality control (QC) of next generation sequencing (NGS) data.   </p>      </li>
      <li>      <p align="left"><a href="http://www.ebi.ac.uk/research/enright/software/kraken" target="_blank"> Kraken</a>  : A set of tools for quality control and analysis of high-throughput sequence data.   </p>      </li>
           </ol>		
           <!--p>&nbsp;</p-->		
      
      </div>
</body>
</html>
