<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title font="Georgia">NGS Resources and Cancer </title>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<link href="menu1.css" type="text/css" rel="stylesheet">
</head>
<body>
<h1 ><font color="#5b1356">Comprehensive Resources for Cancer NGS Data Analysis</font> </h1>
<div id="nav1">
<div id="nav">
<ul>
<li><a href="index.html">HOME</a></li>
<li><a href="ngstool.html">TOOLS</a>
<ul>

<li><a href="cancergenomics.html">Cancer&nbsp;Genomics</a></li>
<li><a href="cancertranscriptomics.html">Cancer&nbsp;Transcriptomics</a></li>
<li><a href="qualitycontrol.html">Quality&nbsp;Control</a></li>
<li><a href="Epigenomics.html">Cancer&nbsp;Epigenomics</a></li>
<li><a href="visualization.html">Cancer&nbsp;Genome&nbsp;visualization</a></li>



</ul>
</li>
<li><a href="pipelines.html">PIPELINES</a>
<ul>
<li><a href="ngsdapipelines.html">Data&nbsp;Analysis</a></li>
<li><a href="ngsvcpipelines.html">Variant&nbsp;Calling</a></li>
<li><a href="ngsqcpipelines.html">Quality&nbsp;Control</a></li></ul>
</li>

</li>
<li><a href="ngscancer.html">NGS and CANCER</a>
<ul>
<li><a href="ngsdataresource.html">Data&nbsp;Resources</a></li>
<li><a href="ngsresources.html">Cancer&nbsp;Databases</a></li>
<li><a href="ngsprojects.html">Cancer&nbsp;Projects</a></li>
<li><a href="pathways.html">Cancer&nbsp;Pathways</a></li>
</ul>
</li>
<li><a href="ngsfaq.html">FAQ</a></li>
<li><a href="ngsteam.html">TEAM</a></li>
<li><a href="ngscontactus.html">CONTACT US</a></li>

</ul>
</div></div><div id="list1">
<p><h2 align="center"><b>Tools in  Visualization</b></h2></p><p align="justify" ><font color="#3f3a01"><b></b></<font></p>
      
  <h3 id="tab-title"> <strong>Data Visualization </strong></h3>
     <ol>   <li>

          <p align="left"><a href="http://www.strand-ngs.com/" target="_blank">Strand NGS</a>: An integrated platform that provides analysis, management and visualization tools for next-generation sequencing data.</p>
        </li>
        <li>
          <p align="left"><a href="http://circos.ca/" target="_blank">CIRCOS</a>: A software package for visualizing NGS data and information.</p>
          
        </li>
        <li>

          <p align="left"><a href="http://www.broadinstitute.org/igv/" target="_blank">IGV</a>: A high-performance visualization tool for interactive exploration of large, integrated  next-generation sequencing data.</p>

        </li>
 <li>      <p align="left"><a href="http://ics.hutton.ac.uk/tablet" target="_blank">Tablet </a> :  A lightweight, high-performance graphical viewer for next generation sequence assemblies and alignments.      </p>      </li>    
 <li>      <p align="left"><a href="http://bamview.sourceforge.net/" target="_blank">Bamview </a> : A free interactive display of read alignments in BAM data files. It has been developed by the Pathogen Group at the Sanger Institute.      </p>      </li>    
 <li>      <p align="left"><a href="http://bioinformatics.bc.edu/marthlab/wiki/index.php/EagleView" target="_blank">EagleView </a> : An information-rich genome assembler viewer with data integration capability.     </p>      </li>    
 <li>      <p align="left"><a href="http://ngsview.sourceforge.net/" target="_blank">NGSView</a> :  A generally applicable, flexible and extensible next-generation sequence alignment editor.      </p>      </li>    
 <li>      <p align="left"><a href="http://bioinfor.com/zoom/lite" target="_blank">ZOOM Lite </a> : A next-generation sequencing data mapping and visualization software.      </p>      </li>    
<!--p>&nbsp;</p-->

     
</ol>
<h3 id="tab-title"> <strong>Genome Browsers/Editors </strong></h3>
<ol>
   <li>
          <p><a href="http://genome.ucsc.edu/" target="_blank">UCSC Genome Browser</a>: A genome browser for any genomic region of specific  interest.</p>
        </li>

 
 <li>      <p align="left"><a href="http://genplay.einstein.yu.edu/wiki/index.php/Main_Page" target="_blank">Genplay </a> : An analyzer and browser for high-throughput data that is being developed by the Stem Cell Genomic Unit at the Albert Einstein College of Medicine of Yeshiva University in New York City.       </p>      </li>
 <li>      <p align="left"><a href="http://genomesavant.com/p/savant/index" target="_blank">Savant </a> :  A next-generation genome browser designed for the latest generation of genome data.       </p>      </li>
 <li>      <p align="left"><a href="http://www.abrowse.org/" target="_blank">ABrowse </a> : A customizable genome browser framework .     </p>      </li>
 <li>      <p align="left"><a href="http://bioviz.org/igb" target="_blank">Integrated Genomic Browser </a> :  An easy-to-use, highly customizable genome browser you can use to view and explore genomic data and annotations, especially RNA-Seq and ChIP-Seq data sets      </p>      </li>
 <li>      <p align="left"><a href="http://www.sanger.ac.uk/resources/software/artemis" target="_blank">Artemis </a> :  A free genome browser and annotation tool that allows visualisation of sequence features, next generation data and the results of analyses within the context of the sequence, and also its six-frame translation.       </p>      </li>
 
 
 </ol>
      <!--p>&nbsp;</p-->
      </div>
</body>
</html>